mos-platform/nes-action53/reset.s

```diff
@@ -1,5 +1,5 @@
 
-.section .reset_stub,"ax",@progbits
+.section .prg_rom_fixed.200,"ax",@progbits
```

Comment:
`.prg_rom_fixed` doesn't have SORT_BY_INIT_PRIORITY set, so using a number as the section name doesn't do anything.  Instead, `.prg_rom_fixed.reset_stub`.